hwdb.d/80-ieee1394-unit-function.hwdb

```diff
@@ -850,6 +850,15 @@ ieee1394:ven000050C2mo00000070sp000050C2ver00000001
  ID_MODEL_FROM_DATABASE=Duende FireWire
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+# The value of model field in unit directory is different from the one in root directory. The value
+# in unit directory is prior to for the modalias.
```

Comment:
I cannot follow the comment. What is 'unit directory'? Is it in another repository?
Please extend the comment, or add a reference about the 'directory'.

Reply:
Indeed. I wrote it under the context of IEEE 1212, so nowadays few developers get it as is... I'll rewrite it.